liver_imaging_analysis/engine/engine.py

```diff
@@ -2,38 +2,51 @@
 a module contains the fixed structure of the core of our code
 
 """
+
 import os
 import random
+from logger import setup_logger
 from liver_imaging_analysis.engine.dataloader import DataLoader, Keys
 import numpy as np
 import torch
 import torch.optim.lr_scheduler
 from liver_imaging_analysis.engine.config import config
 import monai
-from monai.data import Dataset, decollate_batch,  DataLoader as MonaiLoader
+
+from monai.data import Dataset, decollate_batch, DataLoader as MonaiLoader
 from monai.losses import DiceLoss as monaiDiceLoss
+
 from torchmetrics import Accuracy
 from liver_imaging_analysis.engine.utils import progress_bar
 from monai.metrics import DiceMetric, MeanIoU
 import natsort
 from monai.transforms import Compose
 from monai.handlers.utils import from_engine
+from liver_imaging_analysis.engine.tb_tracking import ExperimentTracking
+from torch.utils.tensorboard import SummaryWriter
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
merge approved logger then update this branch

Reply:
done